packages/ui/src/assets/css/input.css

```diff
@@ -0,0 +1,24 @@
+.input-field-wrapper {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+}
+
+.input-field-wrapper > input {
+  border: 1px solid var(--input-border-color);
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+}
+
+.input-field-wrapper > input:focus {
+  outline: 2px solid var(--input-outline-color);
+}
+
+.input-field-wrapper > input:-webkit-autofill {
+  background-color: var(--input-autofill-color);
+}
+
+.input-field-wrapper .input-error-message {
```

Comment:
instead of long .input-error-message it would be better with just .error-message